lib/matplotlib/tests/test_image.py

```diff
@@ -1641,6 +1641,41 @@ def test__resample_valid_output():
         resample(np.zeros((9, 9)), out)
 
 
+@pytest.mark.parametrize("data, interpolation, expected",
+    [(np.array([[0.1, 0.3, 0.2]]), mpl._image.NEAREST,
+      np.array([[0.1, 0.1, 0.1, 0.3, 0.3, 0.3, 0.3, 0.2, 0.2, 0.2]])),
+     (np.array([[0.1, 0.3, 0.2]]), mpl._image.BILINEAR,
```

Comment:
```suggestion
    [(np.array([[0.1, 0.3, 0.2]]), mimage.NEAREST,
      np.array([[0.1, 0.1, 0.1, 0.3, 0.3, 0.3, 0.3, 0.2, 0.2, 0.2]])),
     (np.array([[0.1, 0.3, 0.2]]), mimage.BILINEAR,
```

These values are implemented in a private namespace, but are available in a public namespace (which is already imported directly)

This should also resolve the type hint problems